lib/rspec/support/spec/library_wide_checks.rb

```diff
@@ -46,12 +46,10 @@ def load_all_files(files, preamble, postamble=nil)
   define_method :load_all_lib_files do
     files = all_lib_files - lib_test_env_files
     preamble  = ['orig_loaded_features = $".dup', preamble_for_lib]
-    postamble = [
-      'loaded_features = ($" - orig_loaded_features).join("\n")',
-      "File.open('#{loaded_features_outfile}', 'w') { |f| f.write(loaded_features) }"
-    ]
+    postamble = ['puts ($" - orig_loaded_features).join("\n")']
```

Comment:
This is what's causing the travis failure on older rubies:

```
➜  ~  ruby -v
ruby 1.9.3p448 (2013-06-27 revision 41675) [x86_64-darwin12.4.0]
➜  ~  ruby -w -e "puts ('a' + 'b')"
-e:1: warning: (...) interpreted as grouped expression
ab
```

Ruby doesn't like you having a space between a method name and its arguments when you use parens.  I'd just switch back to the two-line form like we had before.